rootfs/scheduler/k8s.py

```diff
@@ -373,9 +372,6 @@ def _get_service(self, name, namespace):
         return resp.status_code, resp.text, resp.reason
 
     def _create_service(self, name, app_name, app_type):
-        random.seed(app_name)
-        app_id = random.randint(1, 100000)
-        appname = "app-" + str(app_id)
```

Comment:
Ha.  I was looking at this the other day and not liking it either.  Glad to see it go.